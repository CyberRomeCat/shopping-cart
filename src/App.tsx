import HomePage from './components/Homepage';

const App = () => {
  return <HomePage />;
};

export default App;
