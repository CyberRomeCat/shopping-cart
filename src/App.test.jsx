import { render, screen } from '@testing-library/react';

import App from './App';
import { expect } from 'vitest';

describe('Home Page', () => {
  it('renders Home Page', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
