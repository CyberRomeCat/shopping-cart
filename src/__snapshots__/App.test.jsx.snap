// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home Page > renders Home Page 1`] = `
<div>
  <h1>
    Our First Test
  </h1>
</div>
`;
